stepprogress/src/main/java/com/funrisestudio/stepprogress/StepProgressView.kt

```diff
@@ -505,6 +507,42 @@ class StepProgressView : ViewGroup, View.OnClickListener {
         return stepProgress.isAllDone()
     }
 
+    fun setTextTitlePadding(@DimenRes size: Int) {
+        textTitlePadding = resources.getDimension(size)
+        resetView()
+        invalidate()
+    }
+
+    fun setNodeTitleSize(@DimenRes size: Int) {
+        textNodeTitleSize = resources.getDimensionPixelSize(size)
+        resetView()
+        invalidate()
+    }
+
+    fun setTextNodeSize(@DimenRes size: Int) {
+        textNodeSize = resources.getDimensionPixelSize(size)
+        resetView()
+        invalidate()
+    }
+
+    fun setNodeHeight(@DimenRes size: Int) {
+        nodeHeight = resources.getDimension(size)
+        resetView()
```

Comment:
this method should work without resetView()

Reply:
Did you check that one?

Reply:
You were right, this method work without resetView()